src/features/search/atoms/__tests__/Hit.test.tsx

```diff
@@ -13,7 +12,7 @@ import { Hit } from '../Hit'
 const mockSearchState = initialSearchState
 
 const hit = mockedAlgoliaResponse.hits[0]
-const offerId = dehumanizeId(hit.offer.id)
+const offerId = +hit.objectID
```

Comment:
C'est quand même plus explicite non ?
```suggestion
const offerId = Number(hit.objectID) 
```
```suggestion
const offerId = parseInt(hit.objectID) 
```